### PR TITLE
Add @wesselversluis

### PR DESCRIPTION
**Pull request by wesselversluis**

### CODE REVIEW

**Comment by wesselversluis** [comment]
Added "Do you read me?!" assignment 1 from class 2 to the bar chart assignment from class 1.

File: site/wesselversluis/data.tsv [hunk 1]
```diff
@@ -0,0 +1,8 @@
+day	frequency
```
Comment (wooorm) [inline site/wesselversluis/data.tsv]: This looks great, but you’re uploading files to `site/wesselversluis/`, but they should go in `site/class-1-bar/wesselversluis/`.

Comment (wesselversluis) [reply site/wesselversluis/data.tsv]: Ah oké! Wel in mijn eigen repository of op die van de course? Moet ik deze pull request dan verwijderen en even een nieuwe aanmaken?

Comment (wooorm) [reply site/wesselversluis/data.tsv]: It’s probably simplest to remove this branch, making a new branch, and trying once more!

**Comment by wesselversluis** [comment]
Think I did it good this one, my bad.

**Comment by wooorm** [comment]
Most files are in the correct place, but you still have some files in the wrong places: https://github.com/cmda-fe3/course-17-18/pull/189/files

**Comment by wooorm** [comment]
Your bar chart is late, so you won’t receive [points for handing it in on time](https://github.com/cmda-fe3/course-17-18/blob/master/participation.md), but otherwise this won’t affect your participation grade.

Your readme is perfectly fine!

**Pull request merged**